main.py

```diff
@@ -182,3 +185,5 @@
             print(team)
 pygame.quit()
 
+print(time.time()-start)
```

Comment:
dit moet wel ffe netjes dat je weet wat hier staat. Bijv:
```suggestion
print(f"The program took {round(time.time()-start, 2)} seconds to run.")
```